### PR TITLE
doc: update two scheduling algorithm

### PR DESCRIPTION
**Pull request by yfliuuu**
Add description for the IORR (IO sensitive Round Robin) scheduler
and the priority based scheduler.

Signed-off-by: Jie Deng <jie.deng@intel.com>
Signed-off-by: Yifan Liu <yifan1.liu@intel.com>

### CODE REVIEW

**Comment by acrnsi** [comment]
Can one of the admins verify this patch?

**Comment by yfliuuu** [comment]
Continuation of PR #6615

**Review by gvancuts — APPROVED** [review]

File: doc/tutorials/cpu_sharing.rst [hunk 1]
```diff
@@ -99,16 +100,27 @@ and BVT (Borrowed Virtual Time) scheduler.
...
+
+  The IORR (IO sensitive Round Robin) scheduler supports multiple vCPUs running
+  on one pCPU, scheduled by a IO sensitive round robin policy.
+
+- **Priority based scheduler**:
+
+  The priority based scheduler can support vCPU scheduling based on their
+  pre-configured priorities. A vCPU can be running only if there is no
+  higher priority vCPU running on the same pCPU. For example, in some cases,
+  we have two VM, one VM can be configured to use **PRIO_LOW** and the
```
Comment (dbkinder) [inline doc/tutorials/cpu_sharing.rst:113]:
```suggestion
  we have two VMs: one VM can be configured to use **PRIO_LOW** and the
```

File: doc/tutorials/cpu_sharing.rst [hunk 2]
```diff
@@ -99,16 +100,27 @@ and BVT (Borrowed Virtual Time) scheduler.
...
+  other one to use **PRIO_HIGH**, then the vCPU of the **PRIO_LOW** VM can
```
Comment (dbkinder) [inline doc/tutorials/cpu_sharing.rst:114]:
```suggestion
  other one to use **PRIO_HIGH**. The vCPU of the **PRIO_LOW** VM can
```

File: doc/tutorials/cpu_sharing.rst [hunk 3]
```diff
@@ -99,16 +100,27 @@ and BVT (Borrowed Virtual Time) scheduler.
...
+  only be running when the vCPU of the "PRIO_HIGH" VM voluntarily relinquish
```
Comment (dbkinder) [inline doc/tutorials/cpu_sharing.rst:115]:
```suggestion
  only be running when the vCPU of the **PRIO_HIGH** VM voluntarily relinquishes
```

File: doc/tutorials/cpu_sharing.rst [hunk 4]
```diff
@@ -99,16 +100,27 @@ and BVT (Borrowed Virtual Time) scheduler.
...
+  the usage of the pCPU.
```
Comment (dbkinder) [inline doc/tutorials/cpu_sharing.rst:116]:
```suggestion
  usage of the pCPU.
```

**Review by dbkinder — CHANGES_REQUESTED** [review]
Looks good, with a few edits.

**Comment by yfliuuu** [comment]
Re-pushed and comments fixed. @dbkinder

**Review by dbkinder — APPROVED** [review]
Thanks

**Pull request merged**